examples/complete-mssql/main.tf

```diff
@@ -1,33 +1,64 @@
 provider "aws" {
-  region = "us-east-1"
+  region = local.region
 }
 
 locals {
+  name   = "complete-mssql"
+  region = "eu-west-1"
   tags = {
     Owner       = "user"
     Environment = "dev"
   }
 }
 
-##############################################################
-# Data sources to get VPC, subnets and security group details
-##############################################################
-data "aws_vpc" "default" {
-  default = true
-}
+################################################################################
+# Supporting Resources
+################################################################################
+
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.77"
+
+  name = local.name
+  cidr = "10.99.0.0/18"
+
+  azs              = ["${local.region}a", "${local.region}b", "${local.region}c"]
+  public_subnets   = ["10.99.0.0/24", "10.99.1.0/24", "10.99.2.0/24"]
+  private_subnets  = ["10.99.3.0/24", "10.99.4.0/24", "10.99.5.0/24"]
+  database_subnets = ["10.99.7.0/24", "10.99.8.0/24", "10.99.9.0/24"]
 
-data "aws_subnet_ids" "all" {
-  vpc_id = data.aws_vpc.default.id
+  create_database_subnet_group = true
+  enable_nat_gateway           = true
```

Comment:
Please disable NAT gateway to save time during runs.

Reply:
Done